ci/jenkins/test-mc.sh

```diff
@@ -257,14 +261,17 @@ function run_multicluster_e2e {
     export GOCACHE=${WORKDIR}/.cache/go-build
     export PATH=$GOROOT/bin:$PATH
 
+    if [[ ${ENABLE_MC_GATEWAY} ]]; then
```

Comment:
Are we going to have two jobs, one with mc gateway enabled, one with it disabled?

Reply:
yes, I have created a new job for Gateway related change, I suppose we need keep old one to verify those old releases. Old one has an assumption that Pod IPs between member clusters are reachable, so testbed's underlying network are different.